src/ensembl/src/content/app/browser/zmenu/ZmenuController.tsx

```diff
@@ -30,12 +30,9 @@ const ZmenuController = (props: Props) => {
   const [zmenus, setZmenus] = useState<StateZmenu>({});
 
   useEffect(() => {
-    const subscriptionHandler = (payload: ZmenuIncomingPayload) =>
-      handleBpaneEvent(payload);
-
     const subscription = browserMessagingService.subscribe(
       'bpane-zmenu',
-      subscriptionHandler
+      handleBpaneEvent
```

Comment:
_(note for our future selves):_ this change removes unnecessary intermediary function in ZmenuController. It is not directly related to removing the placeholder from the focused BrowserRegionField, but that's fine.